manifests/lib/docs.pp

```diff
@@ -1,5 +1,14 @@
-# This class installs the postgresql-docs See README.md for more
-# details.
+# @summary Installs PostgreSQL bindings for Postgres-Docs. Set the following parameters if you have a custom version you would like to install.
+#
+# @note
+#   Make sure to add any necessary yum or apt repositories if specifying a custom version.
+#
+# @param package_name
+#   String. Specifies the name of the PostgreSQL docs package.
+# @param package_ensure
+#   Enum['present','absent']. Whether the PostgreSQL docs package resource should be present.
```

Comment:
Remove data type. This is not enforced and as such requires a change to the code that should be done in a separate PR. There are other occurrances too throughout.